### PR TITLE
chore: add tests

### PR DESCRIPTION
**Pull request by haimlit**
Includes the following commits:
- chore: add handler tests
- chore: add more construct tests
- chore: fix invalid duration conversion - was detected as a result of failing tests
- chore: remove completed roadmap items

### CODE REVIEW

File: src/drift-monitor.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ export class DriftMonitor extends Construct {
         (props.stackNames === undefined || (props.stackNames!.length === 0))) {
       throw new Error('Must provide at least one stack');
     }
-    if (props.runEvery !== undefined && props.runEvery!.toMinutes() < 1) {
+    if (props.runEvery !== undefined && props.runEvery!.toSeconds() < 60) {
```
Comment (eladb) [inline src/drift-monitor.ts:62]:
```suggestion
    if (props.runEvery !== undefined && props.runEvery.toSeconds() < 60) {
```

File: src/drift-monitor.ts [hunk 2]
```diff
@@ -505,11 +505,11 @@ export class DriftMonitor extends Construct {
       return Duration.minutes(5);
     } else if (duration.toMinutes() <= 15) {
       return Duration.minutes(15);
-    } else if (duration.toHours() <= 1) {
+    } else if (duration.toMinutes() <= 60) {
```
Comment (eladb) [inline src/drift-monitor.ts:508]: Curious why this change was needed?

**Review by eladb — APPROVED** [review]
Nice!

File: src/drift-monitor.ts (hunk 2 above)
Comment (haimlit) [reply src/drift-monitor.ts:508]: `Duration.minutes(59).toHours()` throws `Error: '59 minutes' cannot be converted into a whole number of hours.`. Basically duration throws when the target duration isn't whole numbers.

That means that users may encounter still errors when providing `runEvery: Duration.minutes(61)`, though I'm not sure why users will provide such durations.

File: src/drift-monitor.ts (hunk 1 above)
Comment (haimlit) [reply src/drift-monitor.ts]: TS/TSLint is smart! There are a couple of more places I need to fix then, updated PR.

File: src/drift-monitor.ts (hunk 2 above)
Comment (eladb) [reply src/drift-monitor.ts:508]: I recall some option of Duration to perform integral rounding

Comment (haimlit) [reply src/drift-monitor.ts:508]: Can't believe I've missed that, thanks! [pr#35](https://github.com/cdklabs/cdk-drift-monitor/pull/35) fixes the problem.
